README.md

```diff
@@ -333,228 +333,228 @@ JsonObject jo = Json.createObjectBuilder().add("spi:siteid", "BEDFORD").add("spi
+* To work with child objects, as per the Object Structure definition, you can just make it part of the work order json. The following code shows the creation of a planned labor record or object that is a child of  the work order:
 
 ```java
 JsonObject wplJo = Json.createObjectBuilder().add("skilllevel", "FIRSTCLASS").add("craft", "ELECT").build();
 JsonArray wpLaborArray = Json.createArrayBuilder().add(wplJo).build();
 jo.add("wplabor",wpLaborArray );
 ```
 
-> **note**: the sample uses the lean format.
+> **Note**: The sample uses the lean format.
 
-### 3.2.3 Return with the Properties
-By default, the create operation will not return any content of the new created work order. Since many attribute values get defaulted or auto-generated at the server side based on Maximo business logic, it often makes sense to get the final representation of the newly created resource.
+### 3.2.3 Return with the properties
+By default, the create operation will not return any content of the newly created work order. Because many attributes use default values or calues that auto-generated at the server based on Maximo business logic, it often makes sense to get the final representation of the newly created resource.
 
-Instead of re-selecting the work order again (which makes a round-trip to the server), it is easy to get the resource content in response information when we create a new work order by Maximo Rest Client.
+Instead of reselecting the work order, which makes a round-trip to the server, it is easy to get the resource content in response information when a new work order is created by the Maximo REST Client.
 
-For non-lean ,
+For non-lean:
 
 ```java
 Resource re = rs.create(jo,"spi:wonum", "spi:status","spi:statusdate","spi:description");
 ```
 
-or simply
+or simply:
 		
 ```java
 Resource re = rs.create(jo,"*");
 ```
 
-For lean,
+For lean:
 
 ```java
 Resource re = rs.create(jo,"wonum", "status","statusdate", "description");
 ```
  
-or simply
+or simply:
   
 ```java
 Resource re = rs.create(jo,"*");
 ```
 
-## 3.3 Update a Purchase Order (mxpo)
+## 3.3 Update a purchase order (mxpo)
 
-To update a resource, we can use either the update() or the merge() apis. The difference between the 2 apis is about how they handle the related child objects in the Resource. An example using the PO  Resource (mxpo) will best illustrate the difference. This example will reference two of the Maximo Business Object contained in the resource, the PO(parent) and POLINE(child).
+To update a resource, you can use either the update() or the merge() APIs. The difference between these APIs how they handle the related child objects in the Resource. An example that uses the PO Resource (mxpo) best illustrates this difference. This example will reference two of the Maximo business object that is contained in the Resource: the PO(parent) and POLINE(child).
```

Comment:
line 384-407
the terms PO Line and poline are being used interchangeably, but then there's also the poline array. Recommend that the term PO line should be used consistently and then the poline should only be used for the name of the array.